lib/promptify/railtie.rb

```diff
@@ -57,7 +57,7 @@ def environment
       elsif Rails.env.staging?
         Pry::Helpers::Text.yellow(Rails.env.upcase)
       elsif Rails.env.production?
-        Pry::Helpers::Text.red(Rails.env.upcase)
+        Pry::Helpers::Text.red("#{ProjectHost.get.split('.').first&.upcase} #{Rails.env.upcase}")
```

Comment:
This is an internal dependency. This gem doesn't, and shouldn't, know about the context here. 

Including this `promptify` gem on any other Rails app will fail:

```rb
jcody: /Users/jcody/code/misc/rails_app on master
$ RAILS_ENV=production bundle exec rails c

Loading production environment (Rails 6.1.4)
** [Honeybadger] Reporting error id=d7476d73-e6ff-469b-b776-22910bdf585b level=1 pid=6136
** [Honeybadger] Success ⚡ https://app.honeybadger.io/notice/d7476d73-e6ff-469b-b776-22910bdf585b id=d7476d73-e6ff-469b-b776-22910bdf585b code=201 level=1 pid=6136
/Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/bundler/gems/promptify-a2705d8d4bd5/lib/promptify/railtie.rb:60:in `environment': uninitialized constant Promptify::Railtie::ProjectHost (NameError)
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/bundler/gems/promptify-a2705d8d4bd5/lib/promptify/railtie.rb:48:in `block in new_prompt'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/pry_instance.rb:476:in `block in select_prompt'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/pry_class.rb:377:in `critical_section'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/pry_instance.rb:471:in `select_prompt'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:95:in `read'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:68:in `block in repl'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:67:in `loop'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:67:in `repl'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:38:in `block in start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/input_lock.rb:61:in `__with_ownership'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/input_lock.rb:78:in `with_ownership'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:38:in `start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/repl.rb:15:in `start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/pry-0.14.1/lib/pry/pry_class.rb:188:in `start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/commands/console/console_command.rb:70:in `start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/commands/console/console_command.rb:19:in `start'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/commands/console/console_command.rb:102:in `perform'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/thor-1.1.0/lib/thor/command.rb:27:in `run'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/thor-1.1.0/lib/thor/invocation.rb:127:in `invoke_command'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/thor-1.1.0/lib/thor.rb:392:in `dispatch'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/command/base.rb:69:in `perform'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/command.rb:48:in `invoke'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/railties-6.1.4/lib/rails/commands.rb:18:in `<main>'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
  from /Users/jcody/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.7.5/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
```